src/patternfly/components/InputGroup/examples/input-group-example.hbs

```diff
@@ -0,0 +1,59 @@
+<div class="pf-c-input-group">
+  {{#> button btnClass="pf-m-secondary"}} Button {{/button}}
+  {{#> form-controls controlType="textarea" name="textarea1" id="textarea1" aria-label="textarea example"}}
+{{/form-controls}}
+    {{#> button btnClass="pf-m-tertiary"}} Button {{/button}}
+</div> 
+<br>
+<br>
+<div class="pf-c-input-group">
+  {{#> form-controls controlType="textarea" name="textarea2" id="textarea2" aria-label="textarea example"}}
+{{/form-controls}}
+    {{#> button btnClass="pf-m-tertiary"}} Button {{/button}}
+</div> 
+<br>
+<br>
+<div class="pf-c-input-group">
+{{#> dropdown id="dropdown-example-collapsed"}}Dropdown{{/dropdown}}
+  {{#> form-controls controlType="input" input="true" type="text" id="textInput3" name="textInput3" placeholder="Standard input example" aria-label="standard input example" }}
+{{/form-controls}}
+    {{#> button btnClass="pf-m-secondary"}} Button {{/button}}
+</div>
+<br>
+<br>
+<div class="pf-c-input-group">
+  {{#> form-controls controlType="input" input="true" type="text" id="textInput4" name="textInput4" placeholder="Standard input example" aria-label="standard input example" }}
+  {{/form-controls}}
+  {{#> button btnClass="pf-m-secondary"}} Button {{/button}}
+  {{#> dropdown id="dropdown-example-collapsed"}}Dropdown{{/dropdown}}
+</div>
+<br>
+<br>
+<div class="pf-c-input-group">
+  <span class="pf-c-input-group__text">
+    <i class="fas fa-dollar-sign" aria-label="text input label"></i>
+  </span>
+  {{#> form-controls controlType="input" input="true" type="text" id="textInput5" name="textInput5" placeholder="Standard input example" aria-label="standard input example" }}
+{{/form-controls}}
+<span class="pf-c-input-group__text">
+    .00
+  </span>
+</div>
+<br>
+<br>
+<div class="pf-c-input-group">
+  {{#> form-controls controlType="input" input="true" type="text" id="textInput6" name="textInput6" placeholder="Standard input example" aria-label="standard input example" }}
+{{/form-controls}}
+  <span class="pf-c-input-group__text">
+    @example.com
+  </span>
+</div>
+<br>
+<br>
+<div class="pf-c-input-group">
+  <span class="pf-c-input-group__text">
+    <i class="fas fa-at" aria-label="text input label"></i>
```

Comment:
`aria-label` needs to be on span, as the icon is rendered with aria-hidden="true" same as button example